File: DSA/Yashvardhan-Guliyani/Day12_q1.java
public class Fibonacci {
    public static int fibonacci(int n) {
        if (n <= 0) {
            return 0;
        } else if (n == 1) {
            return 1;
        } else {
            return fibonacci(n - 1) + fibonacci(n - 2);
        }
    }

    public static void main(String[] args) {
        int n = 5;
        System.out.println("The " + n + "-th Fibonacci number is: " + fibonacci(n));
    }
}

File: DSA/saumya/DAY12_Q1.py
def fibonacci(n):
  if n == 0:
    return 0
  elif n == 1:
    return 1
  else:
    return fibonacci(n-1) + fibonacci(n-2)

# TEST CASE-1
n = 5
result = fibonacci(n)
print(f"The {n}th Fibonacci number is: {result}") #output: 5

#TEST CASE-2
n=3
result=fibonacci(n)
print(f"The {n}th Fibonacci number is: {result}") #output: 2
